CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Changelog
 
+# 13.0.0
```

Comment:
I copy-pasted what you wrote in the PR description to start discussing the Changelog :).

The general idea of the Changelog is to be written from the point of view of a OpenFisca Core user (ie someone who is writing their own legislation)